event-publisher/src/main/java/uk/gov/ons/ctp/common/pubsub/PubSubHelper.java

```diff
@@ -369,7 +369,11 @@ private static String buildSubscriberId(EventType eventType) {
     // Use routing key for queue name as well as binding. This gives the queue a 'fake' name, but
     // it saves the Cucumber tests from having to decide on a queue name
     String eventTopicName = eventTopic.getTopic();
-    return eventTopicName + "_cuc";
+    String subSuffix =
+        (eventType.equals(EventType.CASE_UPDATE) || eventType.equals(EventType.UAC_UPDATE))
```

Comment:
Since the RH subscriptions are suffixed `_rh` and the CC subscriptions are suffixed `_cc`  I think we need to change the signatures of the PubSubHelper methods so they know the environment they are operating in before we can decide what suffix to use.   If not, at least put a   "TODO" comment acknowledging this fact.

Reply:
This class will move soon - there is a ticket to do so.
Please change refs to 'queue' to 'topic'
Please add //TODO to parameterise the prefix for the later ticket